hadoop/stackable/patches/3.4.0/005-cyclonedx-plugin.patch

```diff
@@ -0,0 +1,64 @@
+diff --git a/hadoop-dist/pom.xml b/hadoop-dist/pom.xml
+index 6c22e6b..6a657c7 100644
+--- a/hadoop-dist/pom.xml
++++ b/hadoop-dist/pom.xml
+@@ -151,6 +151,24 @@
+           </execution>
+         </executions>
+       </plugin>
++      <plugin>
++        <groupId>org.cyclonedx</groupId>
++        <artifactId>cyclonedx-maven-plugin</artifactId>
++        <version>2.8.0</version>
++        <configuration>
++            <projectType>application</projectType>
++            <schemaVersion>1.5</schemaVersion>
++            <skipNotDeployed>false</skipNotDeployed>
++        </configuration>
++        <executions>
++          <execution>
++            <phase>package</phase>
++            <goals>
++              <goal>makeBom</goal>
++            </goals>
++          </execution>
++        </executions>
++      </plugin>
```

Comment:
Can you check if this is really needed?

https://issues.apache.org/jira/browse/HADOOP-18590

Also: If it is needed, should it use the variable you defined later/earlier in the main pom.xml?

Reply:
It was not needed, I removed it